### DISCUSSION TITLE
Unreal Engine for Carla crashes when just installed latest and run brank UE project

### DISCUSSION THREAD

**Discussion by detectivejieben** [carla-simulator/carla#7573] [Q&A]
Hello, I'm trying to build Unreal Engine for Carla first time.
I'm just trying to do that by following link.
https://carla.readthedocs.io/en/latest/build_windows/

Download latest Unreal Engine from following,
git clone --depth 1 -b carla https://github.com/CarlaUnreal/UnrealEngine.git
and build by Visual studio.
Building finishes successfuly, no error. 

But Unreal Engine crashes when  just run brank UE project.(brank is easier and better to check Unreal Enigne itself)

The error message is following picture.
![image](https://github.com/carla-simulator/carla/assets/168342029/aba910f8-17b4-42f5-8853-269668e9a80c)

[error message]
Assertion fsiled: ResourceTableFrameCounter == INDEX_NONE
[File:C://UnrealEngine/Engine/Source/Runtime/Windows/D3D11RHI/Private/D3D11Texture.cpp][line:2260]

I've checked previous discussion and instaling older version of Unreal Engine for Carla seems the solution.
https://github.com/HARPLab/DReyeVR/issues/90
It was effective and I could run the older Unreal Engine,
but this solution needs also older Carla(0.9.13), and I stacked installing it. 

I would be very thanksful if someome could help me.

**Comment by xavisolesoft** [comment]
Seems this is related with the directX version installed in your machine:
https://github.com/carla-simulator/carla/issues/6201

I suggest you to try running the editor forcing the directX version via flag as they do in the last messages of the above issue link.
Another option would be install a newer version of directX.

**Comment by detectivejieben** [comment]
Thank you so much for your support.
Finally I could open the latest Carla.
I really appreciate for your support!!
